scalding-dagon/src/test/scala/com/twitter/scalding/dagon/MemoizeTests.scala

```diff
@@ -1,8 +1,8 @@
 package com.twitter.scalding.dagon
 
-import org.scalatest.funsuite.AnyFunSuite
+import org.scalatest.FunSuite
 
-class MemoizeTests extends AnyFunSuite {
+class MemoizeTests extends FunSuite {
```

Comment:
our scalatest version for scalding defaults is a bit old so switched to the old API.

Reply:
would probably be a good to upgrade though in general